src/getBullsAndCows.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+'use strict';
+
+const getBullsAndCows = (randomNumber, guessNumber) => {
+  let bulls = 0;
+  let cows = 0;
+  let result = false;
+
+  for (let i = 0; i < randomNumber.length; i++) {
+    if (randomNumber[i] === guessNumber[i]) {
+      bulls++;
+    } else if (randomNumber.includes(guessNumber[i])) {
+      cows++;
+    }
+  }
+
+  if (bulls === 4) {
+    result = true;
+  }
+
+  const resultMessage = result
+    ? 'You won! Congratulations!'
+    : `${bulls} bulls, ${cows} cows`;
+
+  console.log(resultMessage);
```

Comment:
Console logging in the function is not a good practice. It's better to return a result from the function and handle logging elsewhere.